### PR TITLE
Setup esFrontLine on the old cluster

### PR DESCRIPTION
**Pull request by La0**
* Added a script to clone esFrontline and setup its python dependencies in a virtualenv
* Supervisor setup using Python from the virtualenv
* Nginx configuration backported from the new cluster

I tested the scripts & supervisor command line in a fresh Debian docker instance

### CODE REVIEW

**Comment by klahnakoski** [comment]
The  resources/config/nginx.conf changes look good, and it is the only file we must be careful with.  The others.

The contents of active_data.esfrontline.sh should be in https://github.com/mozilla/ActiveData/blob/dev/resources/scripts/setup_frontend_r3_large.sh  if it is wrong, then this service will simply not work.

File: resources/config/supervisord.conf [hunk 1]
```diff
@@ -42,6 +42,17 @@ stdout_logfile=/logs/flask%(process_num)s.log
...
+[program:esFrontLine]
+command=/home/ec2-user/esFrontLine-env/bin/python esFrontLine/app.py --config=resources/config/prod.json
+directory=/home/ec2-user/esFrontLine
+autostart=true
+autorestart=true
+startretries=10
+stderr_logfile=/logs/supervisor_esFrontLine_error.log
+stdout_logfile=/logs/supervisor_esFrontLine.log
+user=ec2-user
+environment=PYTHONPATH="/home/ec2-user/esFrontLine:/home/ec2-user/esFrontLine/vendor:$PYTHONPATH",PYPY_GC_MAX='6GB',HOME='/home/ec2-user'
```
Comment (klahnakoski) [inline resources/config/supervisord.conf]: You probably do not need the PYPY_GC_MAX='6GB'

**Comment by klahnakoski** [comment]
1. perform the nginx changes only
2. reboot nginx (use script to be fast)
3. verify ActiveData still works https://activedata.allizom.org/tools/query.html#query_id=tYU_fyks
4. apply the rest of your changes.  Supervisor can update its config file without having to shutdown, so changes there should not impact the ActiveData service.


The bottom of `setup_frontend_r3_large.sh`  has nginx reboot script:

```bash
# NGINX CONFIG
sudo cp ~/ActiveData/resources/config/nginx_testing.conf /etc/nginx/nginx.conf

# sudo kill -SIGINT `cat /data1/logs/nginx.pid`

sudo /etc/init.d/nginx start

more /data1/logs/nginx.pid
```

**Comment by La0** [comment]
This is now deployed on the old cluster's frontend.

**Pull request merged**